### PR TITLE
Copyediting for 'variable bindings'

### PR DESCRIPTION
**Pull request by steveklabnik**
keeping on with the edits

### CODE REVIEW

File: src/doc/trpl/variable-bindings.md [hunk 1]
```diff
@@ -1,44 +1,48 @@
...
+in the future. If you’re following along, make sure to edit your `main()`
+function, rather than leaving it off. Otherwise, you’ll get an error.
 
-In many languages, this is called a *variable*. But Rust's variable bindings
-have a few tricks up their sleeves. Rust has a very powerful feature called
-*pattern matching* that we'll get into detail with later, but the left
-hand side of a `let` expression is a full pattern, not just a variable name.
-This means we can do things like:
+In many languages, this is called a *variable*, but Rust’s variable bindings
+have a few tricks up their sleeves. For example the left hand side of a `let`
```
Comment (apasel422) [inline src/doc/trpl/variable-bindings.md]: s/left hand/left-hand/

File: src/doc/trpl/variable-bindings.md [hunk 2]
```diff
@@ -1,44 +1,48 @@
...
+book. We don’t need those features for now, so we’ll just keep this in the back
+of our minds as we go forward.
+
+[pattern]: patterns.html
 
 Rust is a statically typed language, which means that we specify our types up
-front. So why does our first example compile? Well, Rust has this thing called
-*type inference*. If it can figure out what the type of something is, Rust
-doesn't require you to actually type it out.
+front, and they’re checekd at compile time. So why does our first example
```
Comment (apasel422) [inline src/doc/trpl/variable-bindings.md]: s/checekd/checked/

File: src/doc/trpl/variable-bindings.md [hunk 3]
```diff
@@ -1,44 +1,48 @@
...
-```{rust}
+```rust
 let (x, y) = (1, 2);
 ```
 
 After this expression is evaluated, `x` will be one, and `y` will be two.
-Patterns are really powerful, but this is about all we can do with them so far.
-So let's just keep this in the back of our minds as we go forward.
+Patterns are really powerful, and have [their own section][pattern] in the
+book. We don’t need those features for now, so we’ll just keep this in the back
```
Comment (tshepang) [inline src/doc/trpl/variable-bindings.md]: the cross-linking here is a nice improvement

**Comment by huonw** [comment]
r=me with @apasel422 notes.

File: src/doc/trpl/variable-bindings.md (hunk 3 above)
Comment (steveklabnik) [inline src/doc/trpl/variable-bindings.md]: Thanks, I think it'll be a big one overall :+1:

**Comment by steveklabnik** [comment]
Thanks @apasel422 !

**Pull request merged**